packages/transaction-controller/CHANGELOG.md

```diff
@@ -6,6 +6,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.0.0]
```

Comment:
It seems that we are missing the functionality added in #1241. I may not have the terminology quite right here but this is as I understood it:

``` suggestion
## [6.0.0]
### Added
- Update transaction controller to automatically initiate, finalize, and cancel approval requests as transactions move through states ([#1241](https://github.com/MetaMask/core/pull/1241))
  - The `ApprovalController:addRequest` event will be emitted when a new transaction is initiated
  - The `ApprovalController:rejectRequest` event will be emitted if a transaction fails to be approved
  - The `ApprovalController:acceptRequest` event will be emitted when a transaction is completed 

```

Reply:
Good point. This seems good except that these are actions not events. I'll adjust the wording slightly to fix that. I've adjusted the descriptions slightly as well:


``` suggestion
## [6.0.0]
### Added
- Update transaction controller to automatically initiate, finalize, and cancel approval requests as transactions move through states ([#1241](https://github.com/MetaMask/core/pull/1241))
  - The `ApprovalController:addRequest` action will be called when a new transaction is initiated
  - The `ApprovalController:rejectRequest` action will be called if a transaction fails
  - The `ApprovalController:acceptRequest` action will be called when a transaction is approved 

```